app/views/liquids/_user_subscription.html.erb

```diff
@@ -0,0 +1,70 @@
+<div class="ltag__user-subscription-tag" id="user-subscription-tag" data-author-username=<%= author_username %>>
+  <div class="ltag__user-subscription-tag__container">
+    <div class="ltag__user-subscription-tag__content" id="content">
+      <div class="ltag__user-subscription__profile-images" id="profile-images">
+        <span class="crayons-avatar crayons-avatar--l mr-2 ltag__user-subscription-tag__author-profile-image">
+          <img class="crayons-avatar__image ltag__user-subscription-tag__author-profile-image" src=<%= author_profile_image %> <%= "#{author_username} profile image" %>>
+        </span>
+        <span class="crayons-avatar crayons-avatar--l mr-2 ltag__user-subscription-tag__subscriber-profile-image">
+          <img class="crayons-avatar__image ltag__user-subscription-tag__subscriber-profile-image">
+        </span>
+      </div>
+      <div class="ltag__user-subscription-tag__cta-text fs-3xl" id="user-subscription-cta-text"><%= cta_text %></div>
+      <div class="ltag__user-subscription-tag__subscription-area" id="subscription-area">
+        <div class="ltag__user-subscription-tag__signed-out" id="subscription-signed-out">
+          <div class="fs-base" id="logged-out-text">You must first sign in to DEV.</div>
+          <button class="crayons-btn" id="sign-in-btn">Sign In</button>
+        </div>
+        <div class="ltag__user-subscription-tag__signed-in" id="subscription-signed-in">
+          <button class="crayons-btn" id="subscribe-btn">Subscribe</button>
+          <div class="fs-s" id="logged-in-text">
+            You'll subscribe with
+            <span class="ltag__user-subscription-tag__subscriber-email"></span>
+            the email address associated with your DEV account. To use a
+            different email address, you can
+            <a href="/settings">update your email address in Settings</a>.
+          </div>
+        </div>
+        <div class="ltag__user-subscription-tag__apple-auth fs-s" id="subscriber-apple-auth">
+          <button disabled class="crayons-btn" id="subscribe-btn">Subscribe</button>
```

Comment:
should this button always be disabled? i didn't see any JS to toggle the `disabled` property here so i'm curious about this

Reply:
Yep! I had 2 choices, toggle the existing `subscribe` button in JS or toss in a second one that's always disabled. It's for this UX:
![image](https://user-images.githubusercontent.com/15987080/85637544-7f25a080-b651-11ea-8c4d-daaa1869b20e.png)
Where the button will never be enabled under the "Apple auth UX". Once a user changes their email, they end up in the "Signed in but not subscribed UX" which has the button enabled.

I tossed in a second button because the "Apple Auth UX" was never mocked up so I wanted to leave the Apple chunk untangled from everything else. Putting in JS to toggle the button for the "Signed in but not subscribed UX" and for "Apple Auth UX" would "join/tangle" them so I didn't go that route.

I'll likely move to the JS button toggle depending on where we land with handling "Apple Auth UX".